corfudb-tools/src/main/java/org/corfudb/browser/CorfuStoreBrowser.java

```diff
@@ -65,6 +73,20 @@
     private final CorfuRuntime runtime;
     private final String diskPath;
 
+    private final ConcurrentMap<String, DescriptorProtos.FileDescriptorProto>
```

Comment:
This is the same as ProtobufDescriptorTable in TableRegistry can we use that instead?

Reply:
not needed if we are resusing the methods from DynamicProtobufSerializer